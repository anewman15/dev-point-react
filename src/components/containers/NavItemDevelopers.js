import { Link } from 'react-router-dom';

function NavItemDevelopers() {
  return (
    <div className="navbar-item">
      <Link className="text-primary-800 font-bold" to="/developers">Developers</Link>
    </div>
  );
}

export default NavItemDevelopers;
