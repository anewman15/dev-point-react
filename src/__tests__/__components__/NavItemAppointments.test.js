import { unmountComponentAtNode } from 'react-dom';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavItemDevelopers from '../../components/containers/NavItemDevelopers';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('NavItemAppointments', () => {
  it('renders the NavItemApointments component to the dom', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <NavItemDevelopers />
        </MemoryRouter>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
