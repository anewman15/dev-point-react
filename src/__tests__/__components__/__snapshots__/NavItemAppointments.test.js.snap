// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItemAppointments renders the NavItemApointments component to the dom 1`] = `
<div
  className="navbar-item"
>
  <a
    className="button is-warning is-fullwidth has-text-white has-text-weight-bold"
    href="/developers"
    onClick={[Function]}
  >
    Developers
  </a>
</div>
`;
